charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.47.0
-version: 25.1.0
+version: 25.1.1
```

Comment:
Thank you, @adamchabin, for your PR. Please, bump the minor number with a new feature.

```suggestion
version: 25.2.0
```

Reply:
Agreed